deploy/kubernetes/base/controller.yaml

```diff
@@ -24,6 +24,10 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+          resources:
+            requests:
+              cpu: "25m"
+              memory: "23Mi"
```

Comment:
Was there some best practice about not setting memory requests?  Or was that for limits?

Reply:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/356#issuecomment-515633418

CPU limit not recommended AFAIK. I think requests seem fine, don't think it will affect throttling just pre-emptability and resource packing.